src/frontend/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -46,13 +40,11 @@ const LoginPage: React.FC<LoginPageProps> = ({
           cookiePolicy={'single_host_origin'}
           isSignedIn={true}
         />
-        {process.env.NODE_ENV === 'development' ? (
-          <LoginDev devSetRole={devSetRole} devFormSubmit={devFormSubmit} />
-        ) : (
-          ''
-        )}
+        {process.env.NODE_ENV === 'development' ? <LoginDev devSetUser={devSetUser} devFormSubmit={devFormSubmit} /> : ''}
```

Comment:
general question for the whole PR, but how secure is this using this process.env.NODE_ENV to check if we're on production/develop?

Reply:
uhhhhhh idk. the real security is on teh backend endpoint which definitely is secure so that's fine. if they try to dev signin on prod it just won't work so idk

Reply:
the only not secure part if they can get around it on the frontend is that the dev login page calls getAllUsers which could be seen as a problem? but we don't restrict that anyways so idk